superset-frontend/src/explore/components/controls/FixedOrMetricControl.jsx

```diff
@@ -107,67 +100,83 @@ export default class FixedOrMetricControl extends React.Component {
     return (
       <div>
         <ControlHeader {...this.props} />
-        <Label onClick={this.toggle}>
-          {this.state.type === controlTypes.fixed && (
-            <span>{this.state.fixedValue}</span>
-          )}
-          {this.state.type === controlTypes.metric && (
-            <span>
-              <span style={{ fontWeight: 'normal' }}>metric: </span>
-              <strong>
-                {this.state.metricValue ? this.state.metricValue.label : null}
-              </strong>
-            </span>
-          )}
-        </Label>
-        <Panel
-          className="panel-spreaded"
-          collapsible
-          expanded={this.state.expanded}
-          onToggle={this.toggle}
+        <Collapse
+          ghost
+          css={css`
+            .ant-collapse-header {
+              border: 0px !important;
+              padding: 0px 0px 8px 0px !important;
```

Comment:
may be there is other way override Ant styles except of `important`... may be create more specific selectors to win style points 💯

Reply:
I know in styled-components also thing like that: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity not sure if it will work here, or no...

Reply:
I've used the `&`*n trick a few times with success. It might get a little out of hand in some cases, but is definitely better than `!important` - it's also worth noting that IDs count for a lot more than classes in CSS selectors, if there are any that can be leveraged.

Reply:
Unfortunately there are no IDs available 😞. To remove `!important` I have to use the whole class hierarchy.